setup.py

```diff
@@ -48,6 +48,7 @@
         't5',
         'tensorflow-datasets',
         'tensorflow-text',
+        'bert-for-tf2'
```

Comment:
While it's in general a great idea, we cannot have this package as a dependency for internal reasons, I'm afraid (sorry about that). So we'll need to stick to doing things more manually, like before.

Reply:
Thank you for your suggestions, I tried to adapt them in new commit. Please note that BertEncoder in trax.data.text_encoder
is basically an adaptation of https://github.com/google-research/bert/blob/master/tokenization.py into trax interface. The reason for that is there are too big difference (even in the definition of punctuation or a whitespace) to make use of trax code in a modular approach. It is used in tf_inputs.tokenize as a vocab as you suggested.

I additionally simplified pipeline so that it uses more TFDS building blocks, adapted tests and removed legacy files

Reply:
I removed this dependency. I'm afraid that in order to use bert tokenizers right now it will be necessary to have some kind of own tokenizer version as i did in BertEncoder. Note that before it was solved by importing FullTokenizer from bert package (it wasn't listed as dependency though), now this isn't possible, because bert doesn't work with tf2.